stories/core/Table.stories.tsx

```diff
@@ -2117,3 +2128,142 @@ HorizontalScroll.decorators = [
     </div>
   ),
 ];
+
+export const Virtualized: Story<Partial<TableProps>> = (args) => {
+  const onClickHandler = (
+    props: CellProps<{ name: string; description: string }>,
+  ) => action(props.row.original.name)();
+
+  const columns = useMemo(
+    () => [
+      {
+        Header: 'Table',
+        columns: [
+          {
+            id: 'name',
+            Header: 'Name',
+            accessor: 'name',
+          },
+          {
+            id: 'description',
+            Header: 'Description',
+            accessor: 'description',
+          },
+          {
+            id: 'click-me',
+            Header: 'Click',
+            width: 100,
+            Cell: (props: CellProps<{ name: string; description: string }>) => {
+              const onClick = () => onClickHandler(props);
+              return (
+                <a className='iui-anchor' onClick={onClick}>
+                  Click me!
+                </a>
+              );
+            },
+          },
+        ],
+      },
+    ],
+    [],
+  );
+
+  const data = useMemo(() => {
+    const size = 100000;
+    const arr = new Array(size);
+    for (let i = 0; i < size; ++i) {
+      arr[i] = {
+        name: `Name${i}`,
+        description: `Description${i}`,
+      };
+    }
+    return arr;
+  }, []);
+
+  return (
+    <Table
+      useVirtualization
+      columns={columns}
+      emptyTableContent='No data.'
+      {...args}
+      style={{ height: 440, maxHeight: '90vh' }}
+      data={data}
+    />
+  );
+};
+
+Virtualized.argTypes = {
+  isLoading: { control: { disable: true } },
```

Comment:
Also disable `data`. In other story too.

Reply:
disabled